### PR TITLE
Add support and security policies.

### PR DESCRIPTION
**Pull request by kuasha420**
## Summary

Addresses issue #3433

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Support
+
+If you encounter an issue with Site Kit, first check our [troubleshooting guide](https://sitekit.withgoogle.com/documentation/fix-common-issues/). For plugin usage questions and compatibility issues with other plugins, please use the [plugin's support forum](https://wordpress.org/support/plugin/google-site-kit). Before opening a new topic, please search the forum for existing topics as someone else has likely reported the issue already.
+
+Find documentation and a blog with news on the [plugin website](https://sitekit.withgoogle.com).
```
Comment (aaemnnosttv) [inline SUPPORT.md:5]: @felixarntz "a blog" sounds a bit off here. How about this?

```suggestion
Find documentation and news on [the official Site Kit website](https://sitekit.withgoogle.com).
```

**Review by aaemnnosttv — COMMENTED** [review]
LGTM, just one suggested tweak to the wording. This came from the ACs so I'll send to @felixarntz for final review 👍

File: SUPPORT.md (hunk 1 above)
Comment (felixarntz) [reply SUPPORT.md:5]: LGTM!

**Review by felixarntz — APPROVED** [review]
Great!

**Pull request merged**